views/partials/header.ejs

```diff
@@ -40,6 +58,60 @@
             </p>
           </div>
         </div>
+
+
+        <div class="rf-header__tools">
+                <div class="rf-search-bar" id="header-search">
+                    <label class="rf-label" for="header-search-input">Label de la barre de recherche</label>
+                    <input class="rf-input" placeholder="Rechercher" type="search" id="header-search-input" name="header-search-input">
+                    <button class="rf-btn" title="Rechercher">
+                        <span>
+                            Rechercher
+                        </span>
+                    </button>
+                </div>
+            </div>
+
       </div>
     </div>
+
+    <div class="rf-container">
+      <nav class="rf-nav" role="navigation" aria-label="Menu principal">
+         <ul class="rf-nav__list">
+            <li class="rf-nav__item rf-nav__item--active">
```

Comment:
`rf-nav__item--active` fait apparaitre ce bouton comme actif. 
Ca serait cool qu'il ne soit actif que quand on est sur la page d'accueil. 
Tu te sens de le faire ?

Reply:
J'ai peur que ça touche à mes limites du templating, mais je vais essayer ! J'ai réussi à faire des for, des if et des else, c'est peut être dans mes cordes 😀

Reply:
Bon, je connais pas ejs mais je connais stackoverflow 😀